Foundation/NSDate.swift

```diff
@@ -174,6 +174,28 @@ extension CFDateRef : _NSBridgable {
     internal var _nsObject: NSType { return unsafeBitCast(self, NSType.self) }
 }
 
+extension NSDate : Comparable { }
+
+public func ==(lhs: NSDate, rhs: NSDate) -> Bool {
+    return lhs === rhs || lhs.compare(rhs) == .OrderedSame
+}
+
+public func <(lhs: NSDate, rhs: NSDate) -> Bool {
+    return lhs.compare(rhs) == .OrderedAscending
+}
+
+public func <=(lhs: NSDate, rhs: NSDate) -> Bool {
+    return lhs < rhs || lhs == rhs
+}
+
+public func >(lhs: NSDate, rhs: NSDate) -> Bool {
+    return lhs.compare(rhs) == .OrderedDescending
+}
+
+public func >=(lhs: NSDate, rhs: NSDate) -> Bool {
+    return lhs > rhs || lhs == rhs
```

Comment:
@tomaskraina, isn't implementing `==` and `<` enough to make it conform to `Comparable`? Anyway, this is the same as pull request #19.